pkgs/top-level/octave-packages.nix

```diff
@@ -33,11 +31,6 @@ makeScope newScope (self:
       inherit computeRequiredOctavePackages;
     };
 
-    wrapOctave = callPackage ../development/interpreters/octave/wrap-octave.nix {
```

Comment:
The file stil exists

Reply:
the file is still used in `pkgs/development/interpreters/octave/default.nix`

this is removed because it is in a `let in` that is not used in the rest of the file